src/store/store.ts

```diff
@@ -558,52 +558,51 @@ export const useStore = defineStore('store', {
     },
 
     /**
-     * Commenting out for now, might be used in future.
      * Whether any correction data has changed (for the purpose of showing the
      * fee summary), ie, does not include:
      * - completing party
      * - detail
      * - certify
      * - staff payment
      */
-    // hasCorrectionDataChanged (): boolean {
-    //   if (this.isBaseCorrectionFiling) {
-    //     return (
-    //       this.hasBusinessNameChanged ||
-    //       this.hasBusinessTypeChanged ||
-    //       this.haveNameTranslationsChanged ||
-    //       this.haveOfficeAddressesChanged ||
-    //       this.havePeopleAndRolesChanged ||
-    //       this.hasShareStructureChanged ||
-    //       this.areProvisionsRemoved ||
-    //       this.haveNewResolutionDates
-    //     )
-    //   }
-
-    //   if (this.isFirmCorrectionFiling) {
-    //     return (
-    //       this.hasBusinessNameChanged ||
-    //       this.hasBusinessStartDateChanged ||
-    //       this.hasNaicsChanged ||
-    //       this.haveOfficeAddressesChanged ||
-    //       this.havePeopleAndRolesChanged
-    //     )
-    //   }
-
-    //   if (this.isCoopCorrectionFiling) {
-    //     return (
-    //       this.hasBusinessNameChanged ||
-    //       this.hasAssociationTypeChanged ||
-    //       this.hasSpecialResolutionMemorandumChanged ||
-    //       this.haveOfficeAddressesChanged ||
-    //       this.havePeopleAndRolesChanged ||
-    //       this.hasSpecialResolutionRulesChanged ||
-    //       this.hasSpecialResolutionResolutionChanged
-    //     )
-    //   }
-
-    //   return false // should never happen
-    // },
+    hasCorrectionDataChanged (): boolean {
+      if (this.isBaseCorrectionFiling) {
+        return (
+          this.hasBusinessNameChanged ||
+          this.hasBusinessTypeChanged ||
+          this.haveNameTranslationsChanged ||
+          this.haveOfficeAddressesChanged ||
+          this.havePeopleAndRolesChanged ||
+          this.hasShareStructureChanged ||
```

Comment:
This getter doesn't seem to be working: I removed a share class and the File and Pay button is still disabled. Please test and fix.

Reply:
Never mind.

Sec...

Reply:
If I delete the last share class then this is an error and I'm blocked from filing with an error. This is correct.

(Same thing if I delete the last director.)

Resolving.